### ISSUE TITLE
Error while importing file with dot e.g jquery.datepicker

### ISSUE THREAD

**Issue by losogudok** [jtangelder/sass-loader#167]
Hi! In my main.scss I have Sass imports and when I try to process it with sass-loader it throws an error. This occurs only when I import file with dot e.g. jquery.dataTable. If I rename it, everything works just fine.

```
ERROR in /opt/projects/my-apps/static/scss/SampleApp/main.scss
 Module build failed: ModuleBuildError: Module build failed: 
 @import "../libs/jquery.dataTables";
 ^
 File to import not found or unreadable: ../libs/jquery.dataTables
 in /opt/projects/my-apps/static/scss/SampleApp/main.scss (line 11, column 9)
 at DependenciesBlock.onModuleBuildFailed (/home/andrey/.nvm/versions/node/v4.1.1/lib/node_modules/webpack/node_modules/webpack-core/lib/NormalModuleMixin.js:315:19)
 at nextLoader (/home/andrey/.nvm/versions/node/v4.1.1/lib/node_modules/webpack/node_modules/webpack-core/lib/NormalModuleMixin.js:270:31)
 at /home/andrey/.nvm/versions/node/v4.1.1/lib/node_modules/webpack/node_modules/webpack-core/lib/NormalModuleMixin.js:292:15
 at context.callback (/home/andrey/.nvm/versions/node/v4.1.1/lib/node_modules/webpack/node_modules/webpack-core/lib/NormalModuleMixin.js:148:14)
 at Object.onRender (/opt/projects/my-apps/static/node_modules/sass-loader/index.js:253:13)
 at /opt/projects/my-apps/static/node_modules/sass-loader/node_modules/async/lib/async.js:868:35
 at _arrayEach (/opt/projects/my-apps/static/node_modules/sass-loader/node_modules/async/lib/async.js:91:13)
 at Object.<anonymous> (/opt/projects/my-apps/static/node_modules/sass-loader/node_modules/async/lib/async.js:867:17)
 at Object.callback (/opt/projects/my-apps/static/node_modules/sass-loader/node_modules/async/lib/async.js:44:16)
 at options.error (/opt/projects/my-apps/static/node_modules/node-sass/lib/index.js:280:32)
 @ ./main.js 3:0-11:2
```

**Comment by jhnns** [comment]
Thx for reporting this. This is probably caused by the "funny" [file resolver](https://github.com/jtangelder/sass-loader/blob/a6cc46f3176499c013ee0d5b565f5392f338a8d9/index.js#L344-L350).

**Comment by jhnns** [comment]
Shipped with `3.1.0`

**Comment by losogudok** [comment]
thanks!
